app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
```

Comment:
are we ok storing the security questions in the session? what if a user has to leave their browser mid-session?

could we store them server-side in a Redis blob instead?